lib/capistrano/wearerequired/tasks/wordpress.rake

```diff
@@ -1,18 +1,19 @@
+require 'shellwords'
+
 namespace :wordpress do
 
   desc <<-DESC
     Install WordPress translations.
   DESC
   task :install_translations do
     next unless fetch(:wp_languages).any?
+    languages = fetch(:wp_languages).shelljoin
 
     on roles(:app) do
       within release_path do
-        fetch(:wp_languages).each do |language|
-          execute :wp, "language core install #{language} 2> /dev/null", raise_on_non_zero_exit: false
-          execute :wp, "plugin list --field=name | xargs -I % wp language plugin install % #{language} 2> /dev/null", raise_on_non_zero_exit: false
-          execute :wp, "theme list --field=name | xargs -I % wp language theme install % #{language} 2> /dev/null", raise_on_non_zero_exit: false
-        end
+        execute :wp, "language core install #{languages} 2> /dev/null", raise_on_non_zero_exit: false
+        execute :wp, "language plugin install --all #{languages} 2> /dev/null", raise_on_non_zero_exit: false
```

Comment:
Maybe we should add `--quiet` so that the table is shown at the end instead of each of the notices.

Reply:
Works for me. Let's also remove the `raise_on_non_zero_exit` option which shouldn't be necessary anymore, see https://github.com/wp-cli/language-command/issues/68.